doc_template/visual_behavior_ophys_nwb.rst

```diff
@@ -0,0 +1,110 @@
+| Stimulus  | Natural Images        | /stimulus/presentation/Natural_Images_Lum_Matched_set_training_2017.07.14    |                                              |
+|           |                       |                                                                              |                                              |
++           +-----------------------+------------------------------------------------------------------------------+----------------------------------------------+
+|           | Stimulus Distribution | /general/task_parameters                                                     |                                              |
+|           |                       |                                                                              |                                              |
++-----------+-----------------------+------------------------------------------------------------------------------+----------------------------------------------+
+| Processed | Motion correction     | /processing/ophys/ophys_motion_correction_x                                  |                                              |
+| data      |                       | /processing/ophys/ophys_motion_correction_y                                  |                                              |
+|           |                       |                                                                              |                                              |
++           +-----------------------+------------------------------------------------------------------------------+----------------------------------------------+
+|           | Image segmentation    | /processing/ophys/image_segmentation                                         |                                              |
+|           |                       |                                                                              |                                              |
++           +-----------------------+------------------------------------------------------------------------------+----------------------------------------------+
+|           | Fluorescence          | /processing/ophys/corrected_fluorescence                                     |                                              |
+|           |                       |                                                                              |                                              |
++           +-----------------------+------------------------------------------------------------------------------+----------------------------------------------+
+|           | dF/F                  | /processing/ophys/dff                                                        |                                              |
++-----------+-----------------------+------------------------------------------------------------------------------+----------------------------------------------+
```

Comment:
Maybe it would be easier to work backwards from the methods that grab data from the NWB file?

For Behavior Sessions:
-  licks
- metadata (including about subject)
- rewards
- running speed
- stimulus_presentations
- stimulus_templates
- task_parameters
- trials

For Behavior Ophys Experiments:
- All of the data sources already mentioned in Behavior Sessions
- cell_specimen_table (segmented ROIs)
- corrected_fluorescence_traces
- dff_traces
- events
- eye_tracking
- max_projection
- motion_correction
- roi_masks
- segmentation_mask_image